src/rdkafka.h

```diff
@@ -81,10 +82,17 @@ typedef SSIZE_T ssize_t;
 #else
 #include <sys/socket.h> /* for sockaddr, .. */
 
+#ifndef __370__
```

Comment:
For clarity I'd like to see this reversed to: ` #ifdef __370__ /* add comment here too */` and then the zos stuff following, after that do the default #else case